packages/node-core/src/indexer/connectionPool.service.ts

```diff
@@ -138,6 +138,10 @@ export class ConnectionPoolService<T extends IApiConnectionSpecific<any, any, an
   }
 
   private getNextConnectedApiIndex(): number | undefined {
+    if (Object.keys(this.pool).length === 0) {
+      throw new Error(`No connected endpoints available`);
```

Comment:
This is ambiguous. I cant tell if it means that all the connected endpoints are available but suspended or we have no connections to any endpoint.